addons/web/static/src/views/fields/date/date_field.xml

```diff
@@ -12,6 +12,8 @@
                 inputId="props.id"
                 placeholder="props.placeholder"
                 onDateTimeChanged="(date) => this.onDateTimeChanged(date)"
+                onInput.bind="onDatePickerInput"
+                onUpdateInput.bind="onUpdateInput"
```

Comment:
can't we use `onDateTimeChanged` props instead?

Reply:
Unfortunately not, onDateTimeChanged is only call if the date was change (so we didn't call it at the begining), and it send a Date in the Luxon format. Here we need to the inputs in string format to compare it correctly with : `ev.target.value`